data/intro missions.txt

```diff
@@ -451,3 +451,39 @@ conversation "end of intro missions"
 	label next
 	`	There's an awkward pause. You're unsure of how to respond. Then he says, "Thing is, this here planet is one of the most exclusive retirement communities in the galaxy. Plenty of the locals are probably worth more than a billion credits. But they're mostly financial or business types, or old money. I'm rich enough to be here, but I just hope they're willing to socialize with someone who made his fortune flying a ship."`
 	`	He hands you a credit chip worth <payment>, and says, "Best of luck to you, Captain. Don't hesitate to look me up if you're ever in the area."`
+
+
+
+mission "Intro Epilogue"
+	minor
+	landing
+	source "New Boston"
+	to offer
+		or
+			"net worth" > 100000
+			has "event: joined the free worlds"
+		not "main plot completed"
+	on offer
+		conversation
+			`The spaceport at <origin> is a completely different experience than you remember. When you first came here, you were a mere child gawking at the spaceships passing through to refuel on their way to bigger and richer worlds; when you left, you were a greenie captain, fighting tooth and nail to remain one step ahead of the bank's repossessors.`
```

Comment:
A spaceport isn't an experience, but visiting one is. So this could be `Visiting the spaceport at <origin> is a completely different experience from the one you remember all those years ago.`

Reply:
Thanks for the review. I added "Visiting" but I didn't add "all those years ago" because the player may have left New Boston less than a year ago.